lib/iiif_manifest/v3/manifest_builder/record_property_builder.rb

```diff
@@ -2,15 +2,17 @@ module IIIFManifest
   module V3
     class ManifestBuilder
       class RecordPropertyBuilder < ::IIIFManifest::ManifestBuilder::RecordPropertyBuilder
-        attr_reader :canvas_builder_factory
+        attr_reader :canvas_builder_factory, :thumbnail_builder_factory
```

Comment:
Similar to above, rename to `:thumbnail_builder`

Reply:
prior art